master/buildbot/config.py

```diff
@@ -127,7 +129,7 @@ def __init__(self):
         "properties", "revlink", "schedulers", "slavePortnum", "slaves",
         "status", "title", "titleURL", "user_managers", "validation", "realTimeServer",
         "analytics_code", "gzip", "autobahn_push", "lastBuildCacheDays",
-        "requireLogin", "globalFactory", "slave_debug_url",
+        "requireLogin", "globalFactory", "slave_debug_url", "slaveManagerUrl",
```

Comment:
There is inconsistency between overall variables naming all over the places. Some of them mixedCase others lower_case. Should we follow some code style in the future?

Also I would call it probably 'agentManagerUrl' ('agent_manager_url') as we discussed to move away from 'slave' names.